### PR TITLE
EWPP-964: Create Task runner command to setup Drupal project local development

### PR DESCRIPTION
**Pull request by ademarco**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+version: '2'
+services:
+  web:
+    image: fpfis/httpd-php-dev:7.2
```
Comment (yenyasinn) [inline docker-compose.yml]:
```suggestion
    image: fpfis/httpd-php-dev:7.3
```

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{
+    "name": "openeuropa/task-runner-drupal-project-symlink",
+    "description": "OpenEuropa TaskRunner command to symlink a Drupal project codebase within a development site, built locally via Composer.",
+    "keywords": ["Robo", "automation", "task-runner", "drupal"],
+    "license": "EUPL-1.2",
+    "type": "library",
+    "minimum-stability": "dev",
+    "prefer-stable": true,
+    "require": {
+        "php": ">=7.2",
```
Comment (yenyasinn) [inline composer.json:10]:
```suggestion
        "php": ">=7.3",
```

File: .drone.yml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+  phpunit:
+    group: test
+    image: ${IMAGE_PHP=fpfis/httpd-php-ci:7.3}
+    commands:
+    - ./vendor/bin/phpunit
+
+matrix:
+  IMAGE_PHP:
+  - fpfis/httpd-php-ci:7.2
```
Comment (yenyasinn) [inline .drone.yml]:
```suggestion
  - fpfis/httpd-php-ci:7.3
```

File: .drone.yml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  - fpfis/httpd-php-ci:7.3
```
Comment (yenyasinn) [inline .drone.yml:52]:
```suggestion
  - fpfis/httpd-php-ci:7.4
```

File: composer.json (hunk 2 above)
Comment (yenyasinn) [inline composer.json]:
```suggestion
        "php": ">=7.3",
```

**Review by yenyasinn — APPROVED** [review]

File: docker-compose.yml [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+  web:
+    image: fpfis/httpd-php-dev:7.3
+    working_dir: /var/www/html
+    ports:
+      - 8080:8080
+    volumes:
+      - .:/var/www/html # Non Mac users.
+      # - nfsmount:/var/www/html # Mac Users with the nfsmount volume.
+    environment:
+      XDEBUG_CONFIG: "remote_enable=1 remote_host=10.254.254.254 remote_port=9000 idekey=PHPSTORM remote_autostart=1"
```
Comment (imanoleguskiza) [inline docker-compose.yml:12]: Lets take the chance to include the new xdebug configuration: https://github.com/openeuropa/multivalue_form_element/pull/7/files

**Review by brummbar — APPROVED** [review]

File: src/TaskRunner/Commands/ProjectSymlinkCommands.php [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+     *
+     * @param string $projectType
+     *   Drupal project type, either 'drupal-module', 'drupal-theme' or 'drupal-profile'.
+     * @return string
+     *   Base directory name, either "modules", "themes" or "profiles".
+     */
+    protected function getProjectTypeDirectory(string $projectType): string
+    {
+        $mapping = [
+            'drupal-module' => 'modules',
```
Comment (imanoleguskiza) [inline src/TaskRunner/Commands/ProjectSymlinkCommands.php:68]: Should this be "modules/custom" and "themes/custom"

Comment (yenyasinn) [reply src/TaskRunner/Commands/ProjectSymlinkCommands.php:68]: Sometimes it is important where we place modules https://github.com/openeuropa/rdf_skos/pull/69/files#diff-fa681d40b4609bc705cae6e98ebb6cf8a1addaf56258af454ed700ea488c0f74R84.

**Pull request merged**